apps/openmw/mwmechanics/aicast.cpp

```diff
@@ -42,7 +42,19 @@ bool MWMechanics::AiCast::execute(const MWWorld::Ptr& actor, MWMechanics::Charac
             return false;
         }
 
-        osg::Vec3f dir = target.getRefData().getPosition().asVec3() - actor.getRefData().getPosition().asVec3();
+        osg::Vec3f targetPos = target.getRefData().getPosition().asVec3();
+        if (target.getClass().isActor())
+        {
+            osg::Vec3f halfExtents = MWBase::Environment::get().getWorld()->getHalfExtents(target);
+            targetPos.z() += halfExtents.z() * 2 * 0.75f;
```

Comment:
Is this definitely supposed to be `... * 2 * 0.75`? The int is just going to get promoted to a float, so it's exactly equivalent to `... * 1.5`, isn't it?

Reply:
We already use such notation. It means "0.75 of actor's height".
If we do not want to use it, I can change it in all cases.

Reply:
I was just making sure it wasn't a typo, for example if you'd meant `... * 2 + 0.75`.